test/integration/auth.spec.ts

```diff
@@ -1801,6 +1892,59 @@ describe('admin.auth', () => {
         });
     });
 
+    it('updateTenant() should not update MFA-related config of tenant when MultiFactorConfig is undefined', () => {
+      expectedUpdatedTenant.tenantId = createdTenantId;
+      const updateRequestNoMfaConfig: UpdateTenantRequest = {
+        displayName: expectedUpdatedTenant2.displayName,
+        multiFactorConfig: undefined,
+      };
+      if (authEmulatorHost) {
+        return getAuth().tenantManager().updateTenant(createdTenantId, updateRequestNoMfaConfig)
+          .then((actualTenant) => {
+            const actualTenantObj = actualTenant.toJSON();
+            // Configuring test phone numbers are not supported in Auth Emulator
+            delete (actualTenantObj as { testPhoneNumbers?: Record<string, string> }).testPhoneNumbers;
+            delete expectedUpdatedTenant2.testPhoneNumbers;
+            expect(actualTenantObj).to.deep.equal(expectedUpdatedTenant2);
+          });
+      }
+      return getAuth().tenantManager().updateTenant(createdTenantId, updateRequestNoMfaConfig)
+        .then((actualTenant) => {
+          expect(actualTenant.toJSON()).to.deep.equal(expectedUpdatedTenant2);
+        });
+    });
+
+    it('updateTenant() should not update SMS MFA related settings when TOTP is disabled', () => {
```

Comment:
nit: can you update the description to "updateTenant() should not disable SMS MFA when TOTP is disabled"

Also, it is slightly inconsistent with the project config tests - there we verify this flow as part of the same test, but we split it here. Just pointing it out, no need to fix it.